src/lib/theme.scss

```diff
@@ -553,6 +553,8 @@
   --rui-modal-size-small-width: 20rem;
   --rui-modal-size-medium-width: 40rem;
   --rui-modal-size-large-width: 60rem;
+  --rui-modal-size-fullscreen-height: 100%;
```

Comment:
I was thinking about "hardcoding" 100% values into _varible.scss but I decided to make it this way - consumer of RUI can override his/her appearance of fullscreen modal.